schutzbot/save_journal.sh

```diff
@@ -2,3 +2,8 @@
 
 # use tee, otherwise shellcheck complains
 sudo journalctl --boot | tee journal-log >/dev/null
+
+# As it might contain sensitive information and is important for debugging
+# purposes, encrypt journal-log with a symmetric passphrase.
+gpg --batch --yes --passphrase "$GPG_SYMETRIC_PASSPHRASE" -o journal-log.gpg --symmetric journal-log
```

Comment:
```suggestion
gpg --batch --yes --passphrase "$GPG_SYMMETRIC_PASSPHRASE" -o journal-log.gpg --symmetric journal-log
```

I made the same comment in the osbuild PR.  The var name spelling isn't that important but it might cause issues when reused.